NEXT_CHANGELOG.md

```diff
@@ -10,5 +10,6 @@
 
 ### Bundles
 * Removed unused fields from resources.models schema: creation\_timestamp, last\_updated\_timestamp, latest\_versions and user\_id. Using them now raises a warning.
+* Fixed default-python template for case when serverless=yes and include\_python=no. ([#2866](https://github.com/databricks/cli/pull/2866))
```

Comment:
```suggestion
* Do not add environments section in default-python templates if there is no python_wheel_task section ([#2866](https://github.com/databricks/cli/pull/2866))
```

Reply:
I prefer original - it tells users what feature is broken(default template with serverless=yes & include_python=no). The stuff about environments and python_wheel_task is impl details.